webapp/tests/Controller/Jury/RejudgingControllerTest.php

```diff
@@ -0,0 +1,60 @@
+<?php declare(strict_types=1);
+
+namespace App\Tests\Controller\Jury;
+
+use App\DataFixtures\Test\RejudgingStatesFixture;
+use App\Tests\BaseTest;
+use Generator;
+
+class RejudgingControllerTest extends BaseTest
+{
+    /**
+     * @dataProvider provideRoles
+     */
+    public function testStartPage(array $roles, int $http): void
+    {
+        $oldRoles = static::$roles;
+        static::$roles = $roles;
+        $this->logOut();
+        $this->logIn();
+        $this->verifyPageResponse('GET', '/jury/rejudgings', $http);
+        if($http===200) {
+            foreach(['No rejudgings defined',' Add new rejudging','Rejudgings'] as $element) {
+                self::assertSelectorExists('body:contains("'.$element.'")');
+            }
+        }
+        // TODO: static::$roles is not always reset between tests
+        static::$roles = $oldRoles;
```

Comment:
Move this to the usual `tearDown` function and then delete here and below.

Reply:
I agree that is a solution, but I think this should be done in the BaseTest instead of the individual tests. I tried it in the BaseTest but some other tests actually seem to rely on this behavior. 

So I prefer to handle this in a separate PR and fix all of them in one go, or did you mean another `tearDown()`?

Reply:
Oh well, if we have tests that rely on this, it's not great, but you're right it's not the problem of this change, so go ahead.